gen/generator.go

```diff
@@ -2224,3 +2236,33 @@ func GetValidValuesToRemove(classPkgName, propertyName string, definitions Defin
 	}
 	return removedValidValuesSlice
 }
+
+func IsInRequiredTestValues(classPkgName, propertyName string, definitions Definitions, testType string) bool {
+	if classDetails, ok := definitions.Properties[classPkgName]; ok {
+		for key, value := range classDetails.(map[interface{}]interface{}) {
+			if key.(string) == "test_values" {
+				for test_type, test_type_values := range value.(map[interface{}]interface{}) {
+					if test_type.(string) == testType {
+						for k := range test_type_values.(map[interface{}]interface{}) {
+							if k.(string) == propertyName {
+								return true
+							}
+						}
+					}
+				}
+
+			}
+		}
+	}
+	return false
+}
+
+func ValidValuesToMap(validValues []interface{}) map[string]string {
```

Comment:
Do we need this function? Or can we store this map instead of the entire validValues from meta in the property struct? This way you can refer to the map directly in the template as ValidValuesMap, and you do not have the change of ValidValues to ValidValuesNames